packages/email-templates/src/templates/eventKeyMined.ts

```diff
@@ -49,5 +49,11 @@ export default {
 <p>It has been added to your <a href="{{keychainUrl}}">Unlock Keychain</a>, where you can view it and, if needed, print it as a signed QR Code!</p>
 
 {{links txUrl openSeaUrl true}}
+
+{{#if transactionReceiptUrl}}
+<strong>Need receipts?</strong>
+<br />
+<span>Access your receipts by visiting <a href="{{transactionReceiptUrl}}">this link</a> you can easily download your transaction receipt.</span>
```

Comment:
can we improve description here? @julien51

Reply:
ok I think we need something simpler like:

```html
<small><a href="{{transactionReceiptUrl}}">Transaction Receipt</a></small>
```

Reply:
Please show a screencapture of the rendered template.

Reply:
**Event key mined** 

<img width="400" alt="Screenshot 2023-05-23 at 15 09 51" src="https://github.com/unlock-protocol/unlock/assets/20865711/3218ae07-1dab-43d4-b78e-0026e8f6a0a7">

**Key mined** 

<img width="400" alt="Screenshot 2023-05-23 at 15 09 39" src="https://github.com/unlock-protocol/unlock/assets/20865711/c9cb0757-f674-4bfa-90cc-62137d3974b8">